services/webnn/ort/graph_builder_ort.cc

```diff
@@ -266,6 +298,13 @@ std::string GraphBuilderOrt::CreateInitializer(base::span<const uint32_t> shape,
   return name;
 }
 
+template <typename DataType>
+  requires internal::IsSupportedTensorType<DataType>
+std::string GraphBuilderOrt::CreateScalarInitializer(const DataType& value) {
+  return CreateInitializer(
+      /*shape=*/{}, base::span(&value, 1u));
```

Comment:
Use `span_from_ref`?
```suggestion
      /*shape=*/{}, base::span_from_ref(value));
```